lib/project.ts

```diff
@@ -488,6 +492,78 @@ export class Project implements Project.IProject {
 		}).future<void>()();
 	}
 
+	private hasConfigurationSpecificDataForProperty(normalizedPropertyName: string) {
+		var result = false;
+		_.forEach(_.keys(this.configurationSpecificData), (config: string) => {
+			if(_.contains(_.keys(this.configurationSpecificData[config]), normalizedPropertyName)) {
+				result = true;
+				// break forEach
+				return false;
+			}
+		});
+
+		return result;
+	}
+
+	private printConfigurationSpecificDataForProperty(normalizedPropertyName: string): void {
```

Comment:
This function should either return the cli-table or even better, the array containing the output data. The caller function should be responsible for printing the data out.